### ISSUE TITLE
How to find the best model when training

### ISSUE THREAD

**Issue by quocnhat** [biubug6/Pytorch_Retinaface#113]
Hi, Thanks very much for sharing your awesome works.
Due to the variation of the dataset ( hard & easy examples in each iteration), the out loss is unstable at all. So how can we determine the best model when training?

**Comment by DerekChen88** [comment]
Same problem. I wonder how to calculate AP during training in order to find the best model. The provided evaluation method seems cannot be used in the training.

**Comment by quocnhat** [comment]
for now, The simplest method is to save the best loss checkpoint crosses the number of epochs. I will evaluate that method when finish.
